apps/web/.env.example

```diff
@@ -132,3 +132,19 @@ AZURE_BLOB_STORE_ACCOUNT_KEY=
 FEATURE_FLAG_BOAS_10_TEST_FEATURE= "true"
 
 
+
+## Development #################################################################
+
+
+# Optional location of a json file containing dummy AD user data when AUTH_DISABLED=true
+# An example of one user in the file is as follows:
+# [
+#   {
+#     "id": "1",
```

Comment:
can this be clear that it will be a guid, eg '1111-2222-3333'

Reply:
I've changed the id to a guid in the example as suggested.